tests/e2e/gc_metadata_syncer.go

```diff
@@ -89,13 +89,11 @@ var _ = ginkgo.Describe("[csi-guest] pvCSI metadata syncer tests", func() {
 		svcClient, svNamespace := getSvcClientAndNamespace()
 		setResourceQuota(svcClient, svNamespace, defaultrqLimit)
 		vcAddress := e2eVSphere.Config.Global.VCenterHostname + ":" + sshdPort
-		if isVsanhealthServiceStopped {
+		ctx, cancel := context.WithCancel(context.Background())
+		defer cancel()
+		if isVsanHealthServiceStopped {
```

Comment:
No default value set in before each for isVsanHealthServiceStopped

Reply:
Done